app/src/main/java/com/aletheiaware/perspective/android/ui/GameActivity.java

```diff
@@ -431,6 +435,10 @@ private String getNextWorld() {
                 if (manager.hasPurchased(PerspectiveAndroidUtils.WORLD_TWELVE)) {
                     return PerspectiveAndroidUtils.WORLD_TWELVE;
                 } // else fallthrough
+            case PerspectiveAndroidUtils.WORLD_TWELVE:
+                if (manager.hasPurchased(PerspectiveAndroidUtils.WORLD_THIRTEEN)) {
+                    return PerspectiveAndroidUtils.WORLD_THIRTEEN;
+                } // else fallthrough
             default:
                 return null;
```

Comment:
FTF: Suggestion for the future return an empty string or throw an exception and then mark routine as @NonNull.

Reply:
Done in Beta 0.8